README.md

```diff
@@ -273,19 +273,38 @@ _Note_: setting up this parameter to more than a few minutes can make the state
 the VM be reported as unresponsive/unavailable on the Azure portal. Also, this 
 setting affects how fast the agent starts executing extensions. 
 
-#### __AutoUpdate.Enabled__
+#### __AutoUpdate.UpdateToLatestVersion__
 
-_Type: Boolean_  
+_Type: Boolean_
 _Default: y_
 
 Enables auto-update of the Extension Handler. The Extension Handler is responsible 
 for managing extensions and reporting VM status. The core functionality of the agent
 is contained in the Extension Handler, and we encourage users to enable this option 
 in order to maintain an up to date version.
 
-On most distros the default value is 'y'.
+_Note_:
+1. This option becomes effective from version 2.10.0.8 onwards.
+2. If AutoUpdate.UpdateToLatestVersion is present, it overrides any value set for AutoUpdate.Enabled (if present).
+3. If AutoUpdate.Enabled is present and set to 'n', we adhere to AutoUpdate.Enabled flag's behavior. Refer to the definition of _AutoUpdate.Enabled_ for additional details.
```

Comment:
When I read this it looks like #3 conflicts with #2
In 2 we say that AutoUpdate.Enabled will be ignored no matter its value if AutoUpdate.UpdateToLatestVersion is present.
In 3 we say that if AutoUpdate.Enabled=n we will ignore AutoUpdate.UpdateToLatestVersion.

These two seem to be conflicting

Reply:
In 2, we already said if AutoUpdate.UpdateToLatestVersion is present. so, 3 is for if it not present. If that's not communicating, let make that clear.

Reply:
Ah I see, yes could we add that clarification to 3